src/filewalker.js

```diff
@@ -89,6 +88,7 @@ async function generateOneLevel(path, settings, depth) {
       // Asynchronously computes the canonical pathname by resolving ".", "..", and symbolic links.
       fs.realpath(path, (err, rpath) => {
         if (err || settings.realPath === false) {
+          console.log('rpath =', rpath);
```

Comment:
I'm thinking we can probably remove this, but we can merge it in and then remove it. :)

Reply:
Also just testing out the ability to comment here on specific lines of code, very cool!